File: src/main/java/upbrella/be/store/repository/StoreMetaRepository.java
package upbrella.be.store.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;
import upbrella.be.store.entity.StoreMeta;

public interface StoreMetaRepository extends JpaRepository<StoreMeta, Long>, StoreMetaRepositoryCustom {

    Optional<StoreMeta> findByClassificationIdAndDeletedIsFalse(long id);
}

File: src/main/kotlin/upbrella/be/store/repository/StoreMetaRepository.kt
package upbrella.be.store.repository

import org.springframework.data.jpa.repository.JpaRepository
import upbrella.be.store.entity.StoreMeta

interface StoreMetaRepository : JpaRepository<StoreMeta, Long>, StoreMetaRepositoryCustom {

    fun existsByClassificationIdAndDeletedIsFalse(classificationId: Long): Boolean
}
